dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/intermediate/components/PercentageComponent.kt

```diff
@@ -41,6 +32,13 @@ class PercentageComponent(
         )
     }
 
+    override fun generateDefaultUploadConfig(uploadCategoryBuilder: UploadCategoryBuilder) {
+        uploadCategoryBuilder.addStandardUploadConfigCell(
+            component = this,
+            uploadComponentName = "PercentageFormField",
```

Comment:
I think we are trying to move away from this upload component, because it is somewhat redundant. The job can be done by a `NumberFormField` with unit set to `%`. I think we did so already in some frameworks for percentage fields.

Reply:
Also where was the upload config before? 😮

Reply:
Ok I can easily remove the PercentageComponent in this story, because there is no field anyways that uses it.
This only exists, because apparently Marc used it in the Heimathafen branch.
So should I delete it?  It wouldn't break anything in this branch here because it is unused.

The uploadconfig was not there before, because noone noticed that it is missing, because no field in GDV uses this component here.